core/execution/pegged_orders.go

```diff
@@ -84,11 +89,25 @@ func (p *PeggedOrders) Park(o *types.Order) {
 	o.Price = num.UintZero()
 	o.OriginalPrice = num.UintZero()
 
-	p.ordersChanged = true
+	p.parked = append(p.parked, o)
+	p.isParked[o.ID] = struct{}{}
+}
+
+func (p *PeggedOrders) Unpark(o *types.Order) {
```

Comment:
I don't think this should be passed an order, ID should be enough

Reply:
Correct. Probably just copied/past another function here to write it.

Reply:
addressed